### PR TITLE
CFE-3388/master: Added example for storejson()

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins, with exotics, please.

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5440/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5440/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5440/

File: examples/storejson.cf [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+{
+  methods: "example_storejson";
+}
+###############################################################################
+#+end_src
+#+begin_src example_output
+#@ ```
+#@ R: {
+#@       "msg": "Hello from globals"
+#@ }
```
Comment (olehermanse) [inline examples/storejson.cf:52]: @nickanderson There is no reports promise for this one?

**Comment by nickanderson** [comment]
@cf-bottom jenkins, with exotics, please.

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/5443/)

(with exotics)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/5443/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-5443/

**Pull request merged**